collectors/python.d.plugin/pandas/README.md

```diff
@@ -0,0 +1,51 @@
+<!--
+title: "Pandas"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/pandas/README.md
+-->
+
+# Pandas Netdata Collector
+
+A python collector using [pandas](https://pandas.pydata.org/) to pull data and do pandas based preprocessing before feeding to Netdata.
+
+## Requirements
+
+This collector depends on some Python (Python 3 only) packages that can usually be installed via `pip` or `pip3`.
+
+```bash
+$ sudo pip install pandas requests
```

Comment:
Should we use `pip` or `pip3`? :thinking: 
Does this collector run on every python version?

Reply:
Probably needs to be python 3. 

It could work with python 2 but pandas not supporting python 2 anymore: https://pandas.pydata.org/pandas-docs/version/0.23/install.html#plan-for-dropping-python-2-7